app/components/one_time_code_input_component.scss

```
@@ -3,11 +3,13 @@
 lg-one-time-code-input {
   display: block;
 
-  @include at-media('tablet') {
-    width: 50%;
-  }
+  
 }
 
 .one-time-code-input__input {
+
+  @include at-media('tablet') {
+    max-width: 50% !important;
+  }
```

We should avoid `!important`. We can make this apply by ensuring that the [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) of the selector is stronger than the one which is taking precedence over this style.

```suggestion
  &.usa-input {
    @include at-media('tablet') {
      max-width: 50% !important;
    }
  }
```

(This is kinda the inverse of the problem described at https://github.com/18F/identity-idp/pull/10394#issuecomment-2049857818 , where we'd probably want our application component stylesheets to be loaded after the default design system components, where currently they have equal specificity, but the default design system styles are winning out because they're loaded later)